tracim/tracim/model/__init__.py

```diff
@@ -60,4 +101,30 @@ def init_model(engine):
 
 # Import your model modules here.
 from tracim.model.auth import User, Group, Permission
-from tracim.model.data import Content
+from tracim.model.data import Content, ContentRevisionRO
+
+
+@event.listens_for(DBSession, 'before_flush')
+def prevent_content_revision_delete(session, flush_context, instances):
```

Comment:
Integrity check: Deletion (``is_delete``flag passed to True in really) of content must be done with ``tracim.model.new_revision`` context manager usage. To prevent failure with it, raise an Exception if delete one.